### DISCUSSION TITLE
Should tiltIndicator include WEO data?

### DISCUSSION THREAD

**Discussion by maurolepore** [2DegreesInvesting/tiltIndicator#129] [Q&A] [answered]
* Does the WEO data change only once a year? 
* Is  it public?
* If so, can we host it in the package and use the latest one as a default?

cc @lindadelacombaz @Tilmon

**Comment by Tilmon** [comment, marked as answer]
Dear @maurolepore @lindadelacombaz , 

WEO data are public so I don't see any problem with hosting it in a package. Same applies for IPR scenarios, which are not yet in the sample data but will be in the real data. 

I don't know how often they change to be honest, but max once a year.
